clang/lib/Driver/ToolChains/SYCL.h

```diff
@@ -15,16 +15,26 @@
 namespace clang {
 namespace driver {
 
+class Command;
+
 namespace tools {
 namespace SYCL {
 
 // Gather command line arguments for backend compilation call
-void TranslateSYCLTargetArgs(Compilation &C,
-              const llvm::opt::ArgList &Args, const ToolChain &TC, llvm::opt::ArgStringList &CmdArgs);
+void TranslateSYCLTargetArgs(Compilation &C, const llvm::opt::ArgList &Args,
+                             const ToolChain &TC,
+                             llvm::opt::ArgStringList &CmdArgs);
 
 // Gather command line arguments for device specific link
-void TranslateSYCLLinkerArgs(Compilation &C,
-              const llvm::opt::ArgList &Args, const ToolChain &TC, llvm::opt::ArgStringList &CmdArgs);
+void TranslateSYCLLinkerArgs(Compilation &C, const llvm::opt::ArgList &Args,
+                             const ToolChain &TC,
+                             llvm::opt::ArgStringList &CmdArgs);
+
+void constructLLVMForeachCommand(Compilation &C, const JobAction &JA,
```

Comment:
Could you please switch `TranslateSYCL*Args` function names to camel case along the way?

Reply:
Maybe changing lines 24-31 with my patch is not right at all, because it can break git blame. What do you think?

Reply:
Fair enough. Stylistic NFC-fixes for the untouched code can always be applied with a separate patch.

Reply:
I leaved it as it was before me. Just because git blame is effective way to explore the code, and confuse it is not right.